frontend/packages/noobaa-storage-plugin/src/components/data-resiliency-card/data-resiliency-card.tsx

```diff
@@ -0,0 +1,138 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { Icon } from 'patternfly-react';
```

Comment:
Please use PF4 icons instead of PF3.  See https://github.com/openshift/console/pull/2054.

Reply:
@rhamilto 
this fixes it: https://github.com/openshift/console/pull/2195